src/main/java/io/jenkins/plugins/gitlabserverconfig/servers/GitLabServer.java

```diff
@@ -249,9 +249,15 @@ public FormValidation doTestConnection(@QueryParameter String serverUrl,
             LOGGER.info("Testing Connection..");
```

Comment:
Suggest adding more context, like `"Testing Connection for GitLab, url:"  + serverUrl`

Reply:
Fixed in 20e31ae.